### PR TITLE
[PR-18472] Add documentation for “Claim by name“ campaign type 

### PR DESCRIPTION
**Pull request by Doimhneacht**
## Demo
https://void-docs.talkable.com/campaigns/campaign_types.html
https://void-docs.talkable.com/campaigns/campaign_types.html#claim-by-name
https://void-docs.talkable.com/campaigns/campaign_placements/claim_by_name.html
https://void-docs.talkable.com/advanced_features/name_sharing.html

## Related Stories

[](https://talkable.atlassian.net/browse/PR-18472)

### CODE REVIEW

**Review by dgtv — APPROVED** [review]
Code LG, but please validate the content with Vova. Thanks

Also, I've noticed that on the screenshot placeholder of "Friends name" looks more like first and last name, not like a "personal" name (nickname) which could be confusing.

File: source/integration/custom/validating_integration.rst [hunk 1]
```diff
@@ -63,6 +63,11 @@ There are 4 types:
 4. :underline:`Advocate Dashboard Standalone:` displays inline in your /referrals
    page
 
+5. :underline:`Claim by Name` displays as a floating widget button on your checkout page.
+
+   .. note::
+      Claim by Name campaign has its own DIV container and can be shown along with the Floating Widget on the same page.
```
Comment (max-si-m) [inline source/integration/custom/validating_integration.rst:69]: I would add that div container is `talkable-claim-by-name`

File: source/integration/custom/validating_integration.rst [hunk 2]
```diff
@@ -63,6 +63,12 @@ There are 4 types:
...
+      The DIV id is `talkable-claim-by-name`.
```
Comment (dgtv) [inline source/integration/custom/validating_integration.rst:70]: Lets specify the HTML code for this:
```html
<div id="talkable-claim-by-name"></div>
```

**Review by dgtv — APPROVED** [review]
Great job!

**Pull request merged**